pkg/controller/commonTestUtils/testUtils.go

```diff
@@ -98,6 +99,38 @@ func NewHyperConvergedConfig() *sdkapi.NodePlacement {
 	}
 }
 
+func NewHyperConvergedConfig2() *sdkapi.NodePlacement {
```

Comment:
The dup's purpose is to create `NodePlacement` instance which is different than the one created by the already existing `NewHyperConvergedConfig()` function. The naming is not great, I agree. I wanted to (re)name them `NewNodePlacement()` and `NewOtherNodePlacement()`, but then preferred to avoid touching lots of test files that are already referencing `NewHyperConvergedConfig()` (as this PR is already quite too large). I could use `NewOtherHyperConvergedConfig()`, which is also not great since this isn't creating a HyperConvergedConfig, but at least is still consistent with the original function.

Reply:
Eventually renamed both to `NewNodePlacement()` and `NewOtherNodePlacement()` - took the toll of touching 7-8 additional "unrelated" files.